protocol/flatbuffers/data_feed/tracker.fbs

```diff
@@ -52,4 +52,10 @@ table TrackerInfo {
     editable: bool;
 
     computed: bool;
+
+    /// A human-friendly name to display as the name of the tracker.
+    display_name: string;
+
+    /// name to display as the name of the tracker set by the user
```

Comment:
Why would the server have authority over what the user chose to name the tracker in a particular application? Makes more sense to me that that state should be handled by the application and not the server

Reply:
we actually don't need custom_name, the idea was that display name is user-changeable!